images/nfs/Dockerfile

```diff
@@ -15,36 +15,26 @@
 #Portions of this file came from https://github.com/kubernetes-incubator/external-storage/blob/master/nfs/deploy/docker/Dockerfile, which uses the same license.
```

Comment:
Update this to the Dockerfile source or remove it

Reply:
updated the source.